Add Android 2.6 blog post (#776)
jellyfin/jellyfin.org: blog/2023/12-12-android-2-6/index.mdx

---
title: Jellyfin Android 2.6
authors:
  - nielsvanvelzen
  - Maxr1998
slug: android-v2.6.0
tags: [release, android]
---

Improvements to the Jellyfin mobile experience are here. A new update for the Jellyfin Android app is ready! Read along to learn everything
new in this update!

\- Niels & Max

{/* truncate */}

## What's new

> The focus of this update is maintenance. We've fixed various bugs and made sure the app works on newer Android versions. Noticable is the

Reviewer: ```suggestion
The focus of this update is maintenance. We've fixed various bugs and made sure the app works on newer Android versions. Noticeable is the
```

improved connection screen that now remembers your previously connected servers, making it a lot easier to switch between them.

<center>
  <img
    src={require('./server-selection.png').default}
    alt='Screenshot of the server selection in the Android app'
    style={{ maxHeight: '600px' }}
  />
</center>

### Full release notes

> The complete list of changes for this release including their respective pull requests can be found on

Reviewer: ```suggestion
The complete list of changes for this release, including their respective pull requests, can be found on
```

organizations, so consider donating if you appreciate their work. A big shout-out to all contributors that made this release possible:

**Jellyfin Team**

- [@Maxr1998](https://github.com/Maxr1998) - Sponsor via [GitHub sponsors](https://github.com/sponsors/Maxr1998)
- [@nielsvanvelzen](https://github.com/nielsvanvelzen) - Sponsor via [GitHub sponsors](https://github.com/sponsors/nielsvanvelzen)

**Other contributors**

- [@timm0e](https://github.com/timm0e)
- [@jmarcet](https://github.com/jmarcet)

We'd also like to thank everyone who reported bugs, provided feedback and participated in beta testing! The feedback we received made this
our most stable release to date.

### Helping out

If you have some experience with Android development or with Kotlin and are interested in contributing yourself, feel free to dive into the
source code [on GitHub](https://github.com/jellyfin/jellyfin-android) and open pull requests.

Alternatively, you can help with translating the app into your language on our
[Weblate](https://translate.jellyfin.org/engage/jellyfin-android/) instance.

## Downloads

> Update your app now to experience all these changes! The app stores will auto-update your Jellyfin app if you're already using the app. For

Reviewer: ```suggestion
Update your app now to check out all these changes! The app stores will auto-update your Jellyfin app if you're already using the app. For
```
